components/bl-signature-pad-component/styles/index.less

```diff
@@ -0,0 +1,90 @@
+@signaturePadThemeColor: @themePrimary;
```

Comment:
shouldn't they have the following prefix `@bl-customComponent-signaturePad-`?

Reply:
Yes, of course